bin/stop

```diff
@@ -3,8 +3,10 @@
 set -e
 
 SCRIPT_DIR="$(dirname "$0")"
+SCRIPT_ACTION="stop"
 
 # Sets $ARWEAVE and $ARWEAVE_* variables
+export SKIP_BUILD=yes
```

Comment:
I don't think you need the `export`. I'm always fuzzy on that, but if you export skip_build I think it might persist in the shell environment after you run the script? If so, then we don't want that. 

e.g. SCRIPT_DIR and SCRIPT_ACTION aren't exported, but are still picked up by the `source arweave.env` line which is all we need for now.